com.reprezen.swagedit.dependencies/META-INF/MANIFEST.MF

```diff
@@ -6,25 +6,26 @@ Bundle-ActivationPolicy: lazy
 Bundle-Version: 0.8.0.qualifier
 Bundle-Vendor: ModelSolv, Inc. d.b.a. RepreZen
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-Description: KaiZen OpenAPI Editor Dependencies
 Automatic-Module-Name: com.reprezen.swagedit.dependencies
-Bundle-ClassPath: lib/com.fasterxml.jackson.core.jackson-annotations-2.9.0.jar,
- lib/com.fasterxml.jackson.core.jackson-core-2.9.8.jar,
- lib/com.fasterxml.jackson.core.jackson-databind-2.9.8.jar,
- lib/com.fasterxml.jackson.dataformat.jackson-dataformat-yaml-2.9.8.jar,
- lib/com.github.fge.btf-1.2.jar,
- lib/com.github.fge.msg-simple-1.1.jar,
- lib/com.github.fge.uri-template-0.9.jar,
- lib/com.github.java-json-tools.jackson-coreutils-1.9.jar,
- lib/com.github.java-json-tools.json-schema-core-1.2.10.jar,
- lib/com.github.java-json-tools.json-schema-validator-2.2.10.jar,
- lib/com.google.code.findbugs.jsr305-3.0.1.jar,
- lib/com.googlecode.libphonenumber.libphonenumber-8.0.0.jar,
- lib/com.reprezen.jsonoverlay.jsonoverlay-4.0.4.jar,
- lib/com.reprezen.kaizen.openapi-parser-4.0.4.jar,
- lib/joda-time.joda-time-2.9.7.jar,
- lib/net.sf.jopt-simple.jopt-simple-5.0.3.jar,
- lib/org.mozilla.rhino-1.7.7.1.jar,
- lib/org.yaml.snakeyaml-1.23.jar
+Bundle-ClassPath: lib/com.fasterxml.jackson.core.jackson-annotations.jar,
+ lib/com.fasterxml.jackson.core.jackson-core.jar,
+ lib/com.fasterxml.jackson.core.jackson-databind.jar,
+ lib/com.fasterxml.jackson.dataformat.jackson-dataformat-yaml.jar,
+ lib/com.github.fge.btf.jar,
+ lib/com.github.fge.msg-simple.jar,
+ lib/com.github.fge.uri-template.jar,
+ lib/com.github.java-json-tools.jackson-coreutils.jar,
+ lib/com.github.java-json-tools.json-schema-core.jar,
+ lib/com.github.java-json-tools.json-schema-validator.jar,
+ lib/com.google.code.findbugs.jsr305.jar,
+ lib/com.googlecode.libphonenumber.libphonenumber.jar,
+ lib/com.reprezen.jsonoverlay.jsonoverlay.jar,
+ lib/com.reprezen.kaizen.openapi-parser.jar,
+ lib/joda-time.joda-time.jar,
+ lib/net.sf.jopt-simple.jopt-simple.jar,
+ lib/org.mozilla.rhino.jar,
+ lib/org.yaml.snakeyaml.jar
```

Comment:
Why don't we need the version suffixes anymore?

Reply:
It is explained here https://github.com/RepreZen/KaiZen-OpenAPI-Editor/pull/518/commits/aa52a267670b8c1cb35b8727b67b6f29b3e0fcec

Shortly build scripts could not update those so we should have versionless names.